tests/Earthfile

```diff
@@ -513,6 +513,8 @@ dockerfile-test:
         --mount=type=tmpfs,target=/tmp/earthly \
         -- --no-output +test
 
+    DO +RUN_EARTHLY --earthfile=dockerfile/privileged.earth --extra_args="--allow-privileged --no-output" --target=+test-allow-privileged-with-privileged-run-command
```

Comment:
I think all that's needed is to change the `+test` on line 514 to `+test-all`, then delete this line.

Reply:
can this line be deleted?

Reply:
Done. Just realized that with the latest fix I don't have to abstract the test anymore :sweat_smile: